.pre-commit-config.yaml

```diff
@@ -14,7 +14,6 @@ repos:
       - id: check-yaml
       - id: end-of-file-fixer
       - id: mixed-line-ending
-      - id: trailing-whitespace
```

Comment:
This incorrectly removed some whitespace inside a multiline string (inside a `Box`), which led to errors.